docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: model
+title: BERT Sequence Classification - Identify Antisemitic texts
+author: John Snow Labs
+name: bert_sequence_classifier_antisemitism
+date: 2021-11-06
+tags: [en, open_source]
+task: Text Classification
+language: en
+edition: Spark NLP 3.3.2
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is imported from `Hugging Face-models and it was trained on 4K tweets, where ~50% were labeled as antisemitic. The model identifies if the text is antisemitic or not. 
+- `1` : Antisemitic
+- `0` : Non-antisemitic
+
+## Predicted Entities
+
+`1`, `0`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/bert_sequence_classifier_antisemitism_en_3.3.2_2.4_1636196636003.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler() \
+    .setInputCol('text') \
+    .setOutputCol('document')
+
+tokenizer = Tokenizer() \
+    .setInputCols(['document']) \
+    .setOutputCol('token')
+
+sequenceClassifier = BertForSequenceClassification \
+      .pretrained('bert_sequence_classifier_antisemitism', 'en') \
+      .setInputCols(['token', 'document']) \
+      .setOutputCol('class')
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([["The Jews have too much power!"]]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler() 
```

Comment:
`\` are these needed?

Reply:
@josejuanmartinez `\` is not needed in scala script.